app/assets/config/manifest.js

```diff
@@ -1,3 +1,3 @@
 //= link_tree ../builds/images
 //= link_tree ../builds/
-//= link introduction_to_supporting_physical_development.jpg
```

Comment:
Rather than adding another link tree here, could we add a task in `package.json`, along with the `postinstall` that installs assets from `govuk-frontend`?

Reply:
Really?

The error I get when this line is missing is:
```
Asset `introduction_to_supporting_physical_development.jpg` was not declared to be precompiled in production.
Declare links to your assets in `app/assets/config/manifest.js`.
```
That implies to me that the manifest.js file is the place rails expects the declaration to be placed. By using `link_tree` it means any file put into `app/assets/images` will be loaded into the asset pipeline. As we will be adding a lot of images, I think that makes sense.

I don't really see how that is worse than the custom copy command used in govuk-frontend which looks pretty hacky to me:
```
cp -R node_modules/govuk-frontend/govuk/assets/fonts/. app/assets/builds/fonts && cp -R node_modules/govuk-frontend/govuk/assets/images/. app/assets/builds/images
```

Reply:
It's putting the image management in one place, rather than having some images that are copied in the build and other s that are linked to directly.  But it was a question on my part, so happy either way.  Why not just put the image directly into the build directory?